test/integration/testpaths/testpath_attach_disk_zwps.py

```diff
@@ -207,3 +208,188 @@ def test_01_attach_datadisk_to_vm_on_zwps(self):
             "Check: Data if Disk is attached to VM")
 
         return
+
+
+class TestAttachDataDiskOnCWPS(cloudstackTestCase):
```

Comment:
is the name of this class a typo? it ends with CWPS (cluster wide primary storage?).